.github/workflows/ci.yml

```diff
@@ -50,23 +50,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: git fetch --tags
+        with:
+          fetch-depth: 0
 
-      - name: Download data
-        env:
-          DATA_PASSWORD: ${{ secrets.DATA_PASSWORD }}
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install doc dependencies
         run: |
-          ./download_test_data.sh
+          sudo apt update --yes && sudo apt install --yes git build-essential pandoc curl graphviz
+          pip install -U pip setuptools wheel
+          pip install -e .[docs]
+          git describe --tags
+          python -c 'import ctapipe; print(ctapipe.__version__)'
```

Comment:
`ctapipe` should be changed in `pyirf` (this is also the reason why the `docs` part of the CI is failing)

Reply:
thx